### ISSUE TITLE
Slashes in syntastic_javac_config

### ISSUE THREAD

**Issue by Scuilion** [Scuilion/gradle-syntastic-plugin#6]
Slashes in syntastic_javac_config should be forward even in Windows

**Comment by rhart92** [comment]
Has there been any progress on making the slashes forward even in Windows? Right now, I have a script that every time the file updates, it changes all the slashes to forward but it would be nice if this was supported.

**Comment by Scuilion** [comment]
What version of the plugin are you using?

**Comment by rhart92** [comment]
0.3.8

**Comment by Scuilion** [comment]
The best option here may be to add a manually configurable switch to set the direction of the slash. Every time I look into this issue I feel like I get turned around. Is it running on cygwin/vim, gvim, or bash for windows. Too many use cases to test for.

**Comment by Dridi** [comment]
Any reason why `string.replace('\\', '/')` wouldn't do the trick? Naive question, not a windows user myself.

**Comment by Scuilion** [comment]
Yeah, the should do the trick. The problem I keep running into is in some configuration on windows (I can't remember now) the slashes need to be *nix style. I don't have a windows box handy. I'll have to see if I can find one and get a little more specific.

**Comment by JohnTJohnston** [comment]
One new vote to support automatic slash manipulation.  I'm using git bash on Windows, and I have confirmed that forward slashes or double backslashes function correctly.
